plugin/evm/gossiper.go

```diff
@@ -62,7 +62,7 @@ type pushGossiper struct {
 
 	// [recentTxs] prevent us from over-gossiping the
 	// same transaction in a short period of time.
-	recentTxs *cache.LRU
+	recentTxs *cache.LRU[common.Hash, interface{}]
```

Comment:
@aaronbuchwald not sure if you prefer `struct{}` here instead of `interface{}` (then we can put struct{}{} instead of nil).

Reply:
Ya let's use `struct{}{}` here